cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -62,6 +62,7 @@ class CrossEntropyDepolarizingModel:
     covariance: Optional[np.ndarray] = None
 
 
+@dataclasses.dataclass
 @protocols.json_serializable_dataclass(frozen=True)
```

Comment:
The line below it, `@protocols.json_serializable_dataclass(frozen=True)`, already turns the class into a dataclass.